sdk/avs/Azure.ResourceManager.Avs/samples/Generated/Samples/Sample_WorkloadNetworkVmGroupCollection.cs

```diff
@@ -20,7 +20,7 @@ public partial class Sample_WorkloadNetworkVmGroupCollection
         public async Task CreateOrUpdate_WorkloadNetworksCreateVMGroup()
         {
             // Generated from example definition: specification/vmware/resource-manager/Microsoft.AVS/stable/2023-09-01/examples/WorkloadNetworks_CreateVMGroup.json
-            // this example is just showing the usage of "WorkloadNetworks_CreateVmGroup" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "WorkloadNetworks_CreateVMGroup" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `VM`: https://github.com/Azure/azure-rest-api-specs/blob/c326a242f0211388079304c119a80aad8c032434/specification/vmware/resource-manager/Microsoft.AVS/stable/2023-09-01/vmware.json#L6276